keps/sig-node/1797-configure-fqdn-as-hostname-for-pods/kep.yaml

```diff
@@ -13,15 +13,19 @@ creation-date: 2020-05-18
 reviewers:
   - "@thockin"
   - "@marosset"
+  - "@dashpole"
```

Comment:
@dashpole is concentrated on instrumentation part of the k8s now. Maybe this name can be used for historical perspective for alpha version review.